### PR TITLE
[WIP] Adding version control (git) support to remix plugin

### PR DESCRIPTION
**Pull request by edisinovcic**
Created profile for version control (git) on remix plugin
Only a subset of commands is currently supported.

### CODE REVIEW

File: projects/utils/src/api/git/api.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { StatusEvents } from '../../types'
+
+export interface IGitSystem {
+  events: {
+    //TODO:
```
Comment (GrandSchtroumpf) [inline projects/utils/src/api/git/api.ts]: As there is no events expected in a git plugin, you can remove TODO

File: projects/utils/src/api/git/profile.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { IGitSystem } from './api'
+import { LibraryProfile } from '../../types'
+
+export const gitProfile: LibraryProfile<IGitSystem> = {
+  name: 'git',
+  methods: ['clone', 'checkout', 'init', 'add', 'commit', 'fetch', 'pull', 'push', 'reset', 'status', 'remote', 'log']
+  //events: ['currentFileChanged']
```
Comment (GrandSchtroumpf) [inline projects/utils/src/api/git/profile.ts]: As there is no events expected in a git plugin, you can remove the comment

File: projects/utils/src/api/git/api.ts (hunk 1 above)
Comment (edisinovcic) [reply projects/utils/src/api/git/api.ts]: This has been deleted

File: projects/utils/src/api/git/profile.ts (hunk 2 above)
Comment (edisinovcic) [reply projects/utils/src/api/git/profile.ts]: This has also been removed

**Pull request merged**